### PR TITLE
Make space_dict global to reuse outside nle.

### PR DESCRIPTION
**Pull request by yobibyte**
I made it a tuple to protect the default object from modification.

### CODE REVIEW

File: nle/env/base.py [hunk 1]
```diff
@@ -37,6 +37,99 @@
 
 SKIP_EXCEPTIONS = (b"eat", b"attack", b"direction?", b"pray")
 
+NLE_SPACE_ITEMS = (
```
Comment (heiner) [inline nle/env/base.py:40]: Any reason for not making this a dict already? Immutability?

Comment (yobibyte) [reply nle/env/base.py:40]: Exactly. Modifying the default object seems dangerous.

File: nle/env/base.py [hunk 2]
```diff
@@ -225,71 +319,7 @@ def __init__(
...
-                low=-15,
-                high=15,
-                **nethack.OBSERVATION_DESC["tty_colors"],
-            ),
-            "tty_cursor": gym.spaces.Box(
-                low=0, high=255, **nethack.OBSERVATION_DESC["tty_cursor"]
-            ),
-        }
-
+        space_dict = dict(NLE_SPACE_ITEMS) if space_dict is None else space_dict
```
Comment (heiner) [inline nle/env/base.py:322]: I think this is fine as the two liner

```suggestion
        if space_dict is None:
            space_dict = dict(NLE_SPACE_ITEMS)
```

Comment (yobibyte) [reply nle/env/base.py]: Wow, github allows committing suggestions.

**Review by heiner — APPROVED** [review]

**Pull request merged**